src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -4279,6 +4303,22 @@ DeletePlacementMetadataCommand(uint64 placementId)
 }
 
 
+/*
+ * AcquireCitusAdvisoryObjectClassLockCommand returns a command to call
+ * citus_internal_acquire_citus_advisory_object_class_lock().
+ */
+char *
+AcquireCitusAdvisoryObjectClassLockCommand(ObjectClass objectClass)
+{
+	StringInfo command = makeStringInfo();
+	appendStringInfo(command,
+					 "SELECT pg_catalog.citus_internal_acquire_citus_advisory_object_class_lock(%d)",
+					 objectClass);
```

Comment:
I wonder if this is more readable when we cast the objectClass from the enum to an int explicitly. It reads weird to format a custom type as %d. Alternatively we could add a macro somewhere like `OBJECTCLASS_FORMAT` in the same manner that other format symbols are aliassed.